data/data/gcp/network/firewall.tf

```diff
@@ -22,12 +22,8 @@ resource "google_compute_firewall" "mcs" {
     ports    = ["22623"]
   }
 
-  source_ranges = var.preexisting_network ? ["0.0.0.0/0"] : [
-    var.network_cidr,
-    google_compute_address.master_nat_ip[0].address,
-    google_compute_address.worker_nat_ip[0].address
-  ]
-  target_tags = ["${var.cluster_id}-master"]
+  source_ranges = var.preexisting_network ? ["0.0.0.0/0"] : [var.network_cidr]
```

Comment:
i think now, it should always be the network_cidr

Reply:
also we need to make sure the source range for health checks for ILB are added here https://github.com/openshift/installer/blob/39c63115aaf52181323d8b76807c3469465969cc/data/data/gcp/network/firewall.tf#L39

Reply:
>also we need to make sure the source range for health checks for ILB are added here

I had not considered this, but it looks like we are covered. Those ranges cover all load balancers except internal https: https://cloud.google.com/load-balancing/docs/health-checks#firewall_rules